ompi/mca/coll/tuned/coll_tuned_alltoall_decision.c

```diff
@@ -115,25 +114,39 @@ int ompi_coll_tuned_alltoall_intra_check_forced_init (coll_tuned_force_algorithm
                                       MCA_BASE_VAR_SCOPE_ALL,
                                       &coll_tuned_alltoall_chain_fanout);
 
-    coll_tuned_alltoall_max_requests = 0; /* no limit for alltoall by default */
+    (void) mca_base_component_var_register(&mca_coll_tuned_component.super.collm_version,
+                                           "alltoall_large_msg",
+                                           "threshold (if supported) to decide if large MSGs alltoall algorithm will be used",
```

Comment:
maybe "use pairwise exchange algorithm for communicators larger than this value"